airbyte-local.sh

```diff
@@ -535,11 +874,13 @@ main() {
     trap cleanup SIGINT
     echo "Created folder $tempdir for temporary Airbyte files"
 
-    if ((no_src_pull)); then
-        log "Skipping pull of source image $src_docker_image"
-    else
-        log "Pulling source image $src_docker_image"
-        docker pull $src_docker_image
+    if [[ -z ${kube_deployment+x} ]]; then
```

Comment:
why did the condition change?

Reply:
in kube deployment we do not do anything locally with docker images (we do not even require to have docker installed on the container where cli runs - so we have to exclude that entire logic when we deploy to kube.